src/Microsoft.AspNetCore.Hosting/Internal/ActivityTrackingOptions.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Hosting.Internal
+{
+    public class ActivityTrackingOptions
```

Comment:
Why not remove the options and stick to the default header names for now? There's no real interoperability story right now anyway since you require the values in a fixed format. It might be better to add options at a later point when it's clearer how the interoperability story will look like.